### DISCUSSION TITLE
Assistance Needed with AI Decks - "Burst Read Failed" Error

### DISCUSSION THREAD

**Discussion by kderakhs** [bitcraze/discussions#1559] [Q&A]
Hello,

We recently purchased five AI decks, but despite following the setup instructions provided on [Bitcraze](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/) website, we’ve been unable to establish a connection. The error we consistently encounter is: "Burst read failed."

Steps We've Taken to Troubleshoot:
Tried multiple debuggers.
Tested different cable connectors.
Swapped between various drones.
Installed the deck both on top and bottom.
Unfortunately, none of these steps have resolved the issue. I've attached some pictures for reference.

Request:
We would appreciate any guidance or suggestions from the community to help us resolve this issue.

Best regards,
![photo_2024-09-20_12-23-31](https://github.com/user-attachments/assets/6e05a48d-d496-4eea-9cc4-031e4df3ba58)
![photo_2024-09-20_12-23-42](https://github.com/user-attachments/assets/83cf1fd2-2a99-4996-ac9c-c3295c138012)
![photo_2024-09-20_12-23-49](https://github.com/user-attachments/assets/a39595c6-c1b2-4878-9972-5cba4ef1582a)
![photo_2024-09-20_12-23-54](https://github.com/user-attachments/assets/f07d20ec-9fe9-4376-aeb1-db363a859354)
![photo_2024-09-20_12-24-00](https://github.com/user-attachments/assets/687e38e1-0d46-423f-99da-03bf68b60efe)

**Comment by gemenerik** [comment]
It looks like you mounted the AI-deck upside down. Check out the [docs](https://www.bitcraze.io/documentation/system/platform/cf2-expansiondecks/#orientation) on how to orient your decks.
